tools/validate/validate.cpp

```diff
@@ -19,9 +19,8 @@ constexpr size_t max_line_length = 120;
 
 class BinaryFile {
 public:
-    explicit BinaryFile(const filesystem::path& filepath) {
-        m_file = _wfopen(filepath.c_str(), L"rb");
-
+    explicit BinaryFile(const filesystem::path& filepath) //

```

Comment:
Why the `//` here? This doesn't look like something clang-format will handle incorrectly.

Reply:
clang-format will put it on the same line; I'm neutral here.

Reply:
This isn't a case that we've overridden clang-format for elsewhere, I'd rather not introduce novelty here and thereby encourage inconsistency. I'll validate and push a change.